tests/python/contrib/test_arm_compute_lib/test_add.py

```diff
@@ -93,6 +93,7 @@ def test_runtime_add():
     for dtype, low, high, atol, rtol, op, op_params in [
         ("float32", -127, 128, 1e-7, 1e-7, relay.add, {}),
         ("uint8", 0, 255, 0.0, 1.0, relay.qnn.op.add, _qnn_params),
+        ("int8", -127, 128, 0.0, 1.0, relay.qnn.op.add, _qnn_params),
```

Comment:
Nit: looks like there has been a mixup between `atol` and `rtol` previously, I'd expect `atol=1` and `rtol=0`

Reply:
Is it the same for `uint8` as well?

Reply:
Yep should be, happy to leave to a followup though

Reply:
I'll update. Thanks